src/mottakerroller.js

```diff
@@ -1,15 +1,19 @@
 const mottakerroller = [
   {
-    kode: 'BRUKER_ELLER_BRUKERS_FULLMEKTIG',
-    term: 'Bruker eller brukers fullmektig'
+    kode: 'BRUKER',
+    term: 'Bruker'
   },
   {
     kode: 'VIRKSOMHET',
     term: 'Virksomheten saken er tilknyttet'
   },
   {
-    kode: 'ARBEIDSGIVER_ELLER_ARBEIDSGIVERS_FULLMEKTIG',
-    term: 'Arbeidsgiver eller arbeidsgivers fullmektig'
+    kode: 'ARBEIDSGIVER',
+    term: 'Arbeidsgiver'
+  },
+  {
+    kode: 'REPRESENTANT',
```

Comment:
Nå er det bare støtte for en fullmektig per sak. Eneste problem ville være hvis det finnes forskjellige fullmektiger for bruker og arbeidsgiver (tror ikke det skjer)

Reply:
Vi har jo støtte for ulike representanter for bruker/arbeidsgiver. 

Men tror ikke det vil bli ett problem siden vi har logikk for "Bruker eller brukers fullmektig" og "Arbeidsgiver eller arbeidsgivers fullmektig" som styre hvilke brev man sender.

Det er jo flere arbeidsgivere som kan ha samme rolle, på lik linje som det kan være flere representanter 🤷‍♀️